mix.exs

```diff
@@ -9,7 +9,16 @@ defmodule Formulator.Mixfile do
      build_embedded: Mix.env == :prod,
      start_permanent: Mix.env == :prod,
      deps: deps(),
-     docs: [source_ref: "#{@version}", extras: ["README.md"], main: "readme"],
+     docs: [
+       source_ref: "#{@version}",
+       extras: [
+         "README.md",
+       ],
+       deps: [
+         ecto: "https://hexdocs.pm/ecto",
```

Comment:
Is this truly a dependency? You don't have to use that feature and the library only requires it in the tests.

Reply:
Their docs say that I should add these configs to ex_docs to link to other libraries' documentation. I can't seem to make this auto-link even when this configured, so apparently I'm doing something wrong. I'm just going to remove it.